addons/l10n_lt/data/account.account.template.csv

```diff
@@ -0,0 +1,184 @@
+"account_account_template_1681","Derivative Financial Assets",1681,"account.data_account_type_non_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_a_3_9","False"
+"account_account_template_171","Deferred Profit Tax Asset",171,"account.data_account_type_non_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_a_4_1","False"
+"account_account_template_2010","Acquisition Cost of Raw Materials, Materials and Assembly Parts",2010,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_1","False"
+"account_account_template_2011","Raw Materials, Materials and Assembly Parts in Transit",2011,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_1","False"
+"account_account_template_2012","Raw Materials, Materials and Assembly Parts at Third Parties",2012,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_1","False"
+"account_account_template_2019","Value Loss of Raw Materials, Materials and Assembly Parts (-)",2019,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_1","False"
+"account_account_template_20200","Cost of Incomplete Production",20200,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_2","False"
+"account_account_template_20210","Cost of Executive Works",20210,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_2","False"
+"account_account_template_2030","Cost of Production",2030,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2035","Production in Transit",2035,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2036","Production at Third Parties",2036,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2039","Value Loss of Production (-)",2039,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_3","False"
+"account_account_template_2040","Cost of Purchased Goods for Resale",2040,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_4","False"
+"account_account_template_2045","Purchased Goods for Resale in Transit",2045,"account.data_account_type_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_b_1_4","False"
```

Comment:
v12 reconciles not only invoices (debts) with payments but also stock movements with invoices (in perpetual warehouse accounting)

2045 is used as both [input](https://github.com/odoo/odoo/pull/35566/files#diff-3725bd6aa589f4df970d5ab5efd1b695R11) and [output](https://github.com/odoo/odoo/pull/35566/files#diff-3725bd6aa589f4df970d5ab5efd1b695R12) accounts. Therefore it should allow reconciliation. 

Example in standard generic COA: https://github.com/odoo/odoo/blob/12.0/addons/l10n_generic_coa/data/account.account.template.csv#L5

Reply:
Good suggestion. Also I think it makes sense to do the same for `2011` and `2035` as those accounts serve same purpose.